src/components/Graph/Graph.tsx

```diff
@@ -18,14 +18,24 @@ interface GraphProps {
   updateEvaluatedPathways: (value: EvaluatedPathway) => void;
 }
 
+const getPath = (pathwayResults: PathwayResults): string[] => {
+  return Object.entries(pathwayResults.documentation)
+    .filter(entry => {
+      const [, doc] = entry;
+      return doc.onPath === true;
+    })
+    .map(entry => {
+      const [, doc] = entry;
+      return doc.state;
+    });
```

Comment:
Since you're never using the keys from these entries, and `doc.onPath` is guaranteed to be a boolean already, can this be simplified to:
```suggestion
  return Object.values(pathwayResults.documentation)
    .filter(doc => doc.onPath)
    .map(doc => doc.state);
```

Reply:
Updated here and in output-results